.rubocop.yml

```diff
@@ -77,7 +77,7 @@ Style/ZeroLengthPredicate:
   Enabled: false
 
 Layout/LineLength:
-  Max: 120
+  Max: 125
```

Comment:
Why was the line length updated?

Reply:
It was reverted.

Reply:
What do you mean?

Reply:
I will put this PR back in draft for now.